tests/core/contracts/test_extracting_event_data.py

```diff
@@ -750,7 +750,7 @@ def test_receipt_processing_with_no_flag(
 
 
 def test_single_log_processing_with_errors(
-        web3,
+        w3,
```

Comment:
We can probably remove these arguments altogether. Doesn't seem like we use the web3 instance in these tests.